pkgs/tools/X11/xdg-utils/default.nix

```diff
@@ -48,6 +49,8 @@ stdenv.mkDerivation rec {
   # just needed when built from git
   nativeBuildInputs = [ libxslt docbook_xml_dtd_412 docbook_xsl xmlto w3m ];
 
+  buildInputs = [ bash ];
```

Comment:
Suggest commenting why this is here to stop someone removing it because "it's in stdenv".